python/ray/data/_default_config.py

```diff
@@ -0,0 +1,2 @@
+# Default file shuffler class to use.
```

Comment:
I plan to move all of default config value (https://github.com/ray-project/ray/blob/master/python/ray/data/context.py#L17-L133) to this file. But that involves more code change because other code paths directly use them. I plan to do it as a separate PR.